tool/tsh/tsh.go

```diff
@@ -267,7 +267,7 @@ func Run(args []string) {
 	ls := app.Command("ls", "List remote SSH nodes")
 	ls.Flag("cluster", clusterHelp).Envar(clusterEnvVar).StringVar(&cf.SiteName)
 	ls.Arg("labels", "List of labels to filter node list").StringVar(&cf.UserHost)
-	ls.Flag("verbose", clusterHelp).Short('v').BoolVar(&cf.Verbose)
+	ls.Flag("verbose", "Enable verbose, one-line output, including node UUIDs").Short('v').BoolVar(&cf.Verbose)
```

Comment:
I would drop 'Enable verbose' , as I would guess it would look like this in the terminal. 

```
  -v, --verbose       Enable verbose, one-line output, including node UUIDs
```

```
  -v, --verbose       One-line output, including node UUIDs
```

Reply:
You're correct:

```
~/go/src/github.com/gravitational/teleport(gus/tsh-help-fix*) » build/tsh ls --help
usage: tsh ls [<flags>] [<labels>]

List remote SSH nodes

Flags:
  -l, --login                    Remote host login
...
  -v, --verbose                  One-line output, including node UUIDs

Args:
  [<labels>]  List of labels to filter node list

Aliases:
```